test/integration/integration.js

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+const find = require('find');
+const utils = require('./utils');
+const setup = require('./setup');
+
+const broadcasting = process.env.BROADCASTING !== 'false';
+const TOTAL_PEERS = Number.parseInt(process.env.TOTAL_PEERS) || 10;
+// Full mesh network with 2 connection for bi-directional communication
+const EXPECTED_OUTOGING_CONNECTIONS = (TOTAL_PEERS - 1) * TOTAL_PEERS * 2;
```

Comment:
I suggest to put all `ENV` variables used in integration tests here. Just for reference, so we can refer back and check which are used for which purpose. Currently their usage is scattered among different files.